services/backend/src/router/profile/validator.js

```diff
@@ -147,6 +147,11 @@ const updateProfileValidator = [
     .withMessage(
       "must be an array of containing { proficiency: 'ORAL' | 'WRITING' | 'READING', level: 'A' | 'B' | 'C' | 'E' | 'X' | 'NA', date?: DateTime }"
     ),
+  body("qualifiedPools")
+    .optional()
+    .isArray()
+    .custom((array) => array.every((i) => isUUID(i.classificationId)))
+    .withMessage("must be an array of containing { classificationId: UUID }"),
```

Comment:
I think it should be more robust, you check for the `selectionProcessNumber`, `jobPosterLink`, `jobTitle`

Reply:
updated